terraform/examples/io500/config/GCP-32C-10S16d-NOGVNIC-rf2.sh

```diff
@@ -55,13 +47,13 @@ DAOS_CLIENT_MACHINE_TYPE=c2-standard-16
 DAOS_CLIENT_GVNIC=false
 
 # Storage
-DAOS_POOL_SIZE="$(awk -v disk_count=${DAOS_SERVER_DISK_COUNT} -v server_count=${DAOS_SERVER_INSTANCE_COUNT} 'BEGIN {pool_size = 350 * disk_count * server_count / 1000; print pool_size"TB"}')"
+DAOS_POOL_SIZE="$(awk -v disk_count=${DAOS_SERVER_DISK_COUNT} -v server_count=${DAOS_SERVER_INSTANCE_COUNT} 'BEGIN {pool_size = 375 * disk_count * server_count / 1000; print pool_size"TB"}')"
```

Comment:
I got no enough space error with "375" in some previous tests.

Reply:
This worked for me when I ran `start.sh -c config/<config>` with each of the `config/GCP*.sh` config files. 

Just to be sure, I ran this 32C-10S16d configuration again and it was fine.

Reply:
Sounds good. Thank you!